workspace.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/sh
+#
+#copyright agloo in 2016 under the WTFPL 
+#constructs and groups windows into workspaces to switch between using your favorite keybinder.
+
+NUM_WS=9
+
+help() {
+	cat << EOF
+usage: $(basename $0) [-hinp] [-g ws_num] [-m ws_num] 
+	-h: Displays this message
+	-i: Initializes workspaces and puts you in workspace 0. Should be called once in a startup script.
+	-n: Move up one workspace
+	-p: Move down one workspace
+	-g, --ws_num: go to the workspace specified by ws_num
+	-m, --ws_num: move the currently focused window to the worskpace specified by ws_num
```

Comment:
The `--something` notation is for long arguments. To make your help more readable, I personally think it should be something like:

    -g <ws_num>: go to the workspace specified by <ws_num>